cmd/fever/cmds/run.go

```diff
@@ -487,6 +487,18 @@ func mainfunc(cmd *cobra.Command, args []string) {
 		}).Info("Flow extraction disabled")
 	}
 
+	// Heartbeat injector
+	haveHeartbeat := viper.GetBool("heartbeat.enable")
```

Comment:
heartbeatEnabled instead of haveHeartbeat for consistency maybe?

Reply:
ok, sure!